src/util/localStorage.native.ts

```diff
@@ -0,0 +1,26 @@
+import _ from 'lodash'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+import { IStorage } from '../types'
+
+let storage: IStorage = {
+  async clear(): Promise<void> {
+    await AsyncStorage.clear()
+  },
+
+  async getItem(key: string): Promise<string | null> {
+    return AsyncStorage.getItem(key)
+  },
+
+  async removeItem(key: string): Promise<void> {
+    storage = _.omit(storage, key)
```

Comment:
`_.omit` is O(n). Can you do `delete storage.key` or do you need a new object reference?

Reply:
Refactored using `delete`. didn't use it earlier because eslint was complaining 😅